webpack/assets/javascripts/react_app/components/HostDetails/Audits/constants.js

```diff
@@ -0,0 +1,2 @@
+export const NUMBER_OF_RECORDS = 3;
+export const BASE_URL = 'api/audits';
```

Comment:
I'm personally not a big fan of using constants in a separate file when there is just one place using them, it makes following the code more difficult.